inventory.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 Mender Software AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package main
+
+type InventoryAttribute struct {
+	Name  string      `json:"name"`
+	Value interface{} `json:"value"`
```

Comment:
`[]string{"foo"} ` becomes `["foo"]` in JSON, while `string(foo)` encodes to `"foo"`. Given the outcome of yours and @eysteins discussion, multiple entries are wrapped into a list. On the other side of the system, the API expects attributes to come in this format `{"name": "foo", "value": <valid JSON>}`. Attributes appearing multiple times will just overwrite it's previous value. Eventually, the client can either send everything as a list, or try to preserve the 'format' you came up with.

I have no opinion here, best if you discuss that with @eysteins @maciejmrowiec and other interested parties. It would be nice if you decide on something in soon enough for me to update the PR, otherwise we'll have another sprint with incomplete tasks.